src/commissaire_service/storage/__init__.py

```diff
@@ -107,19 +110,19 @@ def register_store_handler(self, config):
         self._manager.register_store_handler(
             handler_type, config, *matched_types)
 
-    def _build_model(self, model_type_name, model_kwargs):
+    def _build_model(self, model_type_name, model_json_data):
```

Comment:
I would. It will keep me from needing to wrap every handler call with json.dumps of parameters :smile:. Thanks!